x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/hooks/use_endpoint_action_items.tsx

```diff
@@ -27,7 +27,8 @@ import { isIsolationSupported } from '../../../../../../common/endpoint/service/
  * @param endpointMetadata
  */
 export const useEndpointActionItems = (
-  endpointMetadata: MaybeImmutable<HostMetadata> | undefined
+  endpointMetadata: MaybeImmutable<HostMetadata> | undefined,
+  isEndpointList?: boolean
```

Comment:
This is ok, but I would suggest having these kind of boolean flags in an object, for example:

```Typescript
type options = {
  isEndpointList?: boolean
}
```

So you can then call this hook with:

```Typescript
const endpointActions = useEndpointActionItems(endpointMetadata, { isEndpointList: true });
```

And you will always know what you are enabling. I think it improves the readability but as I said, it is just a suggestion 🙂